commons/src/main/java/org/frankframework/util/PropertyLoader.java

```diff
@@ -49,8 +49,8 @@ public PropertyLoader(ClassLoader classLoader, String propertiesFile) {
 		load(classLoader, propertiesFile);
 
 		//Make sure to not call ClassUtils when using the root instance, as it has a static field referencing to AppConstants
```

Comment:
Erik zegt: hier mag wel een spatie.

Reply:
Ik kan dat niet afdwingen in de instellingen, maar heb wel een spatie gezet

Reply:
Maar wel in de checkstyle.xml volgens mij :)